sale_promotion_rule/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,18 @@
+This module adds the concept of promotion rules that can be applied on the sale
+order.
+
+Two kinds of rules are implemented:
+ * automatic
+ * coupon
+
+Automatic rules are applied/recomputed automatically for a sale order when the
+user clicks on the button "Apply discount" in the view form. Depending on the
+rule's criteria more than one automatic rule can be applied to a same sale
+order.
+
+Coupon are special manual rules. Only one coupon can be applied to a
+sale order. This rule takes always precedence over automatic rules.
+
+
+.. image:: https://raw.githubusercontent.com/OCA/sale-workflow/10.0/sale_promotion_rule/static/description/promotion_rule.png
```

Comment:
You can now write : ../sale_promotion_rule/static/description/promotion_rule.png

(more simple, and make the url resilient, during the migration)

Not a blocking point.

Reply:
thank you @legalsylvain Change done